themes/godotengine/pages/privacy-policy.htm

```diff
@@ -0,0 +1,187 @@
+    <li>The right to restrict our use of your personal data;</li>
+    <li>The right to object to our use of your personal data;</li>
+    <li>The right to receive your personal data in a usable electronic format and transmit it to a third party (also known as the right of data portability); and</li>
+    <li>The right to lodge a complaint with your local data protection authority.</li>
+  </ul>
+
+  <p>If you would like to exercise any of these rights, you may do so by mailing <a href="mailto:privacy@godotengine.org">privacy@godotengine.org</a>. Please understand, however, the rights enumerated above are not absolute in all cases.</p>
+
+  <p>Where the GDPR applies, you also have the right to withdraw any consent you have given to uses of your personal data. If you wish to withdraw consent that you have previously provided to Godot, you may do so by mailing <a href="mailto:privacy@godotengine.org">privacy@godotengine.org</a>. However, the withdrawal of consent will not affect the lawfulness of processing based on consent before its withdrawal.</p>
+
+  <p>Godot will undertake best efforts to provide these rights to people outside of the EEA as well.</p>
+
+  <h3 class="title">How to Access, Modify or Update Your Information</h3>
+
+  <p>Godot gives you the ability to access, modify or update your personal data at any time. On sites where you can create accounts (Q&amp;A, Asset Library), you may log in and make changes to your login information (change your password), your contact information, your general preferences and your personalization settings. If necessary, you may also contact us and describe the changes you want made to the personal data you have previously provided by mailing <a href="mailto:privacy@godotengine.org">privacy@godotengine.org</a>.</p>
+
+  <p>If you wish to remove your personal data from Godot, you may contact us at <a href="mailto:privacy@godotengine.org">privacy@godotengine.org</a> and request that we remove this information from the Godot System. Other locations where you may have used your personal data as an identifier (e.g. mailing list and forum postings in the archives, repository changelogs, and IRC logs) will not be altered.</p>
+
+  <h3 class="title">How to Contact Us</h3>
+
+  <p>If you have any questions about any of these practices or Godot's use of your personal information, please feel free to contact us by email at:</p>
+
+  <p><a href="mailto:privacy@godotengine.org">privacy@godotengine.org</a></p>
+
+  <p>Godot will work with you to resolve any concerns you may have about this Statement.</p>
+
+  <h3 class="title">Changes to this Privacy Statement</h3>
+
+  <p>Godot reserves the right to change this policy from time to time. If we do make changes, the revised Privacy Statement will be posted on this site. A notice will be posted on our homepage for 30 days whenever this privacy statement is changed in a material way.</p>
+
+  <p>This Privacy Statement was last amended on 2020-06-30.</p>
```

Comment:
We could link to https://github.com/godotengine/godot-website/commits/master/themes/godotengine/pages/privacy-policy.htm so people can view diffs between each revision. (The URL will only be valid once this PR is merged.)

Reply:
Ah indeed, I thought about this and then forgot to do it.

Reply:
Added.